app/controllers/users_controller.rb

```diff
@@ -93,6 +93,14 @@ def impersonate
     end
   end
 
+  def invalidate_jwt_for_all_users
+    @users = User.authorized(:edit_users).except_hidden.uniq
+    JwtSecret.where(user_id: @users).destroy_all
+    process_success(
+      :success_msg => _('Successfully invalidated registration tokens for all users.')
```

Comment:
@Lennonka text review please

Reply:
I don't think what would be required because it's in line with other PRs.